src/gui/filedetails/sharemodel.cpp

```diff
@@ -405,6 +402,7 @@ void ShareModel::slotSharesFetched(const QList<SharePtr> &shares)
     }
 
     handlePlaceholderLinkShare();
+    Q_EMIT sharesChanged();
```

Comment:
maybe this should be the last line of `ShareModel::handlePlaceholderLinkShare`
that would make sure it is not omitted by its callers

Reply:
Done